src/form-element-creator.ts

```diff
@@ -55,6 +40,42 @@ export class FormElementCreator {
         return button;
     }
 
+    createPhoneInput = async (form: HTMLFormElement, element: any) => {
+        const paragraph = this.elementCreator.createElement('div');
+        paragraph.textContent = element.name;
+        form.appendChild(paragraph);
+        const select = this.elementCreator.createElement('select');
+        const input = this.elementCreator.createElement('input') as HTMLInputElement;
+        this.setInputAttributes(input, element);
+
+        const countryTelMap = await this.countryApi();
+
+        countryTelMap.forEach((diallingCode: string, countryName: string) => {
+            const option = this.elementCreator.createElement('option') as HTMLOptionElement;
+            option.text = `${countryName}`;
+            option.value = `${diallingCode}`;
+            select.appendChild(option);
+        });
+
+        form.appendChild(select);
+        form.appendChild(input);
+    };
+
+    private async countryApi(): Promise<Map<string, string>> {
```

Comment:
wymysl lepsza nazwe dla tej funkcji. Bo ona nic mi nie mowi. Jest to rzeczownik a pamietaj ze nazwa funkcji to powinno byc wyrazenie rzeczownikowe lub czasownik